### ISSUE TITLE
[Bug] Ryujinx shader cache screen not updating unless mouse is moving

### ISSUE THREAD

**Issue by Bentheminernz** [Ryujinx/Ryujinx#5872]
### Description of the issue

Most of the time during the shader compilation screen that appears when booting a game, the screen doesn't update unless the mouse is moving. This did stop for a while between 1.1.1065 - 1.1.1067 but now it is back. It is most common though when using an external monitor which is odd.

Recorded on 1st november
https://github.com/Ryujinx/Ryujinx/assets/65065992/a6f42f7c-eb8a-46ec-a894-452e4ce96a3c

Recorded on 23rd october
https://github.com/Ryujinx/Ryujinx/assets/65065992/722c7467-91b6-4168-a40b-40afc2eedfe6

### Reproduction steps

1. Start loading a game with a longish shader compilation time, Super Mario Odyssey and Zelda Tears of the Kingdom in my case
2. The screen shouldn't update when compiling shaders

### Log file

[Ryujinx_1.1.1068_2023-11-01_19-09-48.log](https://github.com/Ryujinx/Ryujinx/files/13224030/Ryujinx_1.1.1068_2023-11-01_19-09-48.log)

### OS

MacOS Sonoma 14.2 23C5030f Dev Beta 1 (Occurs on 14.1)

### Ryujinx version

1.1.1068

### Game version

1.3.0

### CPU

Apple M1

### GPU

Apple M1 7-core

### RAM

8GB Unified Memory

### List of applied mods

Super Mario Odyssey 1080p (Occurs without mods)
[SMO 1080p Mod.zip](https://github.com/Ryujinx/Ryujinx/files/13224039/SMO.1080p.Mod.zip)

**Comment by gdkchan** [comment]
For me that happens with everything on macOS, not just the shader compilation screen (for example, the frame rate counter on the bottom doesn't update unless I'm moving the mouse or pressing some key). Seems to be sort of random though, sometimes it is fine. And that is on a macbook, not connected to any external monitor.

**Comment by Bentheminernz** [comment]
Yeah it is kinda of random. After I reported the bug I went back to playing odyssey and it loaded just fine

**Comment by sschamp** [comment]
For me it happens as well, but only when I have alt-tabbed out of Ryujinx.
